lib/components/schematic-box.ts

```diff
@@ -17,10 +17,14 @@ export const schematicBoxProps = z
     paddingBottom: distance.optional(),
 
     title: z.string().optional(),
-    titlePosition: nine_point_anchor.default("top_left"),
+    titleAnchorPosition: z.object({
+      x: position.optional(),
+      y: position.optional(),
+    }),
+    titleAnchorAlignment: nine_point_anchor.default("center"),
     titleColor: z.string().optional(),
     titleFontSize: distance.optional(),
-    titleInside: z.boolean().default(false),
+    titleInside: z.boolean().default(true),
```

Comment:
I prefer the original default because the user doesn’t need to add padding for it to look okay by default